tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "declarationMap": true,
     "pretty": true,
     "moduleResolution": "node",
-    "target": "es5",
+    "module": "commonjs",
+    "target": "es2015",
```

Comment:
Needed for `tsc` to output CommonJS module format...
```
    "module": "commonjs",
```

Needed for JavaScript private properties (e.g., #session)...
```
    "target": "es2015",
```

Reply:
I don't necessarily know if this will affect any apps importing this package, but support for es2015 should be mostly there for node 14 (which is going EOL soon and is our latest supported version), so I don't expect the target will be a big problem.